playbooks/openshift-node/private/restart.yml

```diff
@@ -23,13 +23,8 @@
       name: "{{ item }}"
       state: started
     with_items:
-    - etcd_container
```

Comment:
Not sure what the ideal approach for this file is.  This looks like it's currently broken.

Reply:
I think we can get rid of this whole task.